_docs/tooling/maven.md

```diff
@@ -49,7 +49,8 @@ Here is the list of properties supported by the goal:
 
 * **mainClass** &ndash; a fully qualified name of a class that contains the `main(String[])` method. 
   When this property specified, TeaVM produces `main` function in global scope that runs the translated
-  version of the the `main(String[])` method in the browser.
+  version of the the `main(String[])` method in the browser. There is no default value for this parameter.
```

Comment:
I don't think there's need in `There is no default value for this parameter` here.